kpi/models/import_export_task.py

```diff
@@ -738,8 +755,16 @@ def _b64_xls_to_dict(base64_encoded_upload):
     else:
         survey_dict = xls2json_backends.xls_to_dict(xls_with_renamed_sheet)
         survey_dict['library'] = survey_dict.pop('survey')
+
     return _strip_header_keys(survey_dict)
 
+def _append_kobo_locking_profiles(
+    base64_encoded_upload: BytesIO, survey_dict: dict
+) -> None:
+    decoded_str = base64.b64decode(base64_encoded_upload)
```

Comment:
kind of pedantic, but in the interest of clarity: `b64decode()` returns a `bytes`, not a `str`